packages/contracts/src/RelayHub.sol

```diff
@@ -42,6 +42,7 @@ contract RelayHub is IRelayHub, Ownable {
     }
 
     function setConfiguration(RelayHubConfig memory _config) public override onlyOwner {
+        require(_config.devInfo.fee < 100, "dev fee too high");
```

Comment:
We may consider having more than 2 decimal points for percentage. I would go as far as, say, supporting 1/100'000th as a dev fee.

Reply:
If that's the case, we may as well consider that for relayer fe

Reply:
I guess we should have. What do you think?

Reply:
Not sure honestly, since the trade off is clear between granularity and convenience. It really depends on how much we really need this precision when more relayers/clients are joining the pool.